lib/openstack_query/managers/server_manager.py

```diff
@@ -26,168 +27,114 @@ class ServerManager(QueryManager):
     def __init__(self, cloud_account: CloudDomains):
         QueryManager.__init__(self, query=ServerQuery(), cloud_account=cloud_account)
 
-    def search_all_servers(self, output_details: QueryOutputDetails) -> QueryReturn:
+    def search_all(self, **kwargs) -> QueryReturn:
         """
         method that returns a list of all servers
-        :param output_details: A dataclass containing config info on how results should be returned
+        :param kwargs: A set of optional kwargs to pass to the query
         """
```

Comment:
We should document what the accepted kwargs are, or at least link to the signature rather than leave a future dev cursing your name

Reply:
that was my issue with changing this to `kwargs` - link would make sense